src/main/haskell/kore/test/Test/Kore/Step/Simplification/DomainValue.hs

```diff
@@ -60,7 +64,33 @@ test_domainValueSimplification =
                 )
             )
         )
+    , testCase "\\bottom propagates through builtin Map"
+        (assertEqualWithExplanation
+            "Expected \\bottom to propagate to the top level"
+            (OrOfExpandedPattern.make [])
+            (evaluate
+                mockMetadataTools
+                (DomainValue
+                    testSort
+                    (BuiltinDomainMap (Map.fromList [(Mock.aConcrete, bottom)]))
+                )
+            )
+        )
+    , testCase "\\bottom propagates through builtin List"
```

Comment:
Why not a test for sets?

Reply:
Set keys are concrete values, so they cannot contain `\bottom`.